src/common/components/ui/PopoverIcon.tsx

```diff
@@ -25,6 +25,7 @@ const PopoverIcon: React.FC<PopoverIconProps> = ({
 }) => (
   <Popover {...props}>
     <Button
+      style={{ padding: 0 }}
```

Comment:
Maybe we should only set top/bottom padding and leave the left/right padding as is.
See the attached screenshot.